releng/release-tool/release-tool.go

```diff
@@ -772,7 +772,7 @@ func (r *releaseData) getReleases() ([]*github.RepositoryRelease, error) {
 	return r.allReleases, nil
 }
 
-func (r *releaseData) autoDetectData(autoReleaseCadance string, autoPromoteAfterDays int) error {
+func (r *releaseData) autoDetectData(autoReleaseCadance string, autoPromoteAfterDays int) error { //nolint:gocyclo
```

Comment:
Are these nolints here just to get something running initially?

Reply:
yes, also the git grep is to remind us to remove them